### ISSUE TITLE
Not able to view zip/tar files on Linux

### ISSUE THREAD

**Issue by klutchnikoff** [fman-community/fman#323]
Hi!

Similarly to #251, the file pane does not display anything when I press the "enter" key on a **tar** or a **zip** file. However, the correct URL `tar:///path/to/my/file/foo.tar` is displayed.

FYI i'm using Ubuntu 17.10 and here are some basic tests that I made in my terminal:
```
$ which 7za
/usr/bin/7za
```
and
```
$ 7za l -ba -slt foo.tar
Path = bar.txt
Folder = -
Size = 561
Packed Size = 1024
Modified = 2018-02-08 22:23:11
Mode = -rwxrwxrwx
User = 
Group = 
Symbolic Link = 
Hard Link = 
```

**Comment by mherrmann** [comment]
Sorry for the trouble. I have an idea what could be causing this. I'll take a look.

**Comment by klutchnikoff** [comment]
Hi,

I just realized that the problem only happens in specific situations. When file names include accented letters (for example).

I hope this can help.

Nicolas.

**Comment by mherrmann** [comment]
Actually, that does help! Thanks. I'm deep in the trenches of some backend work for fman. I'll take a look when I'm done with this.

**Comment by mherrmann** [comment]
Fixed in [fman 0.9.6](https://fman.io/changelog#v0.9.6). If you can still reproduce it with some files I may have not tested, please let me know.
